presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -24,6 +24,13 @@ As an example, on a 11-node cluster dedicated to Presto (1 Coordinator + 10 Work
   * `query.initial-hash-partitions = 10`
   * `task.concurrency = 8`
 
+If your workload consists of queries that have very quick processing time of the splits you might encounter situation when whole cluster is under utilized (very small usage of CPU, network and disks).
+For example this can happen when your queries are highly selective and most splits can be filtered out just by looking into theirs headers (like bloom filters or min/max statistics in ORC files) without reading the actual data.
+In such case you might want to increase values of the following properties:
```

Comment:
why? what for?

Reply:
sorry, didnt understand your question. You mean why this particular troubleshooting tip?

Reply:
There is a case described above and there is a suggestion to user what should be tuned, but there is no explanation how this change is going influence the system. It would be nice to say to user how this will help.